tests/TestHelpers/HttpLogger.php

```diff
@@ -0,0 +1,126 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Sajan Gurung <sajan@jankaritech.com>
+ * @copyright Copyright (c) 2017 Sajan Gurung sajan@jankaritech.com
```

Comment:
Someone's living in the past 😉 
```suggestion
 * @copyright Copyright (c) 2023 Sajan Gurung sajan@jankaritech.com
```